Microsoft.DotNet.Interactive/Extensions/SerializerExtensions.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.IO;
+using Newtonsoft.Json;
+
+namespace Microsoft.DotNet.Interactive.Extensions
```

Comment:
The `Extensions` namespace refers to kernel extensions, not extension methods.

Serialization stuff might make more sense in the `Server` namespace.